### ISSUE TITLE
Автоматическое включение/выключение db_update

### ISSUE THREAD

**Issue by a-KaDeMi-k** [Limych/GeniBase#77]
Подумал тут...
Вот сейчас вроде бы все записи у нас проиндексированы. Т.е. получается, что ежеминутное срабатывание в cron.php функции db_update (хотя бы в части индексации новых фамилий) - это бесполезная трата ресурсов и загрузка сервера лишними запросами.

Может можно придумать какой-нибудь флаг включения (при появлении новых данных в person_raw) и флаг отключения (когда запрос, отбирающий фамилии для индексации перестаёт возвращать данные)?

**Comment by Limych** [comment]
Да, подобное возможно. Но чуть игаче: если обновлять нечего, система автоматом переключится в режим, когда обновление будет вызываться, скажем, раз в 100 запусков крона. А как появятся данные, автоматом включится обычный режим.

Но для этого нужно сделать систему хранения опций: #25
